### PR TITLE
Fixing height of discussion input

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

closes https://github.com/forem/forem/issues/9782

## QA Instructions, Screenshots, Recordings

1. Go to an article in dev.to
2. Scoll down to disicussion input
3. Focus on it
4. The height will remain same

## Added tests?
- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/assets/stylesheets/comments.scss [hunk 1]
```diff
@@ -413,7 +413,7 @@ a.header-link {
         color: var(--form-placeholder);
       }
       &.embiggened {
-        height: calc(2vw + 115px);
+        height: calc(2vw + 106px);
```
Comment (Rafi993) [inline app/assets/stylesheets/comments.scss:416]: I changed this value to **106px** from **115px** after experimenting with it making sure it does not change. But I couldn't explain why **106px** fixes it and why it was **115px** before (I couldn't find any reference. Is it sum of height of some elements?).

File: app/assets/stylesheets/comments.scss [hunk 2]
```diff
@@ -413,7 +413,7 @@ a.header-link {
         color: var(--form-placeholder);
       }
       &.embiggened {
-        height: calc(2vw + 115px);
+        height: 125px;
```
Comment (Rafi993) [inline app/assets/stylesheets/comments.scss:416]: I can't explain on why ```125px``` fixed this (or why it was ```2w + 115px``` before). I got this value after trial and error.

**Comment by benhalpern** [comment]
I think the fact that this changes is more a bug than a feature, but the issue where it changes inconsistently and doesn't grow as you type is more a bug.

But we probably don't want to change this until we merge a future adjustment to the comment design system as a whole.

**Comment by ludwiczakpawel** [comment]
Yeap, I have a PR work in progress where I addressed this issue (alongside other design updates im workin on for comments). @Rafi993 Is it ok with you if we close this one?

**Comment by Rafi993** [comment]
> Yeap, I have a PR work in progress where I addressed this issue (alongside other design updates im workin on for comments). @Rafi993 Is it ok with you if we close this one?

Awesome. Do close this one.

**Comment by Rafi993** [comment]
Since you have PR feel free to change the owner of the issue https://github.com/forem/forem/issues/9782

**Pull request closed without merging**